### PR TITLE
Release notes for 20.12 update 1

### PR DESCRIPTION
**Pull request by iansk**

### CODE REVIEW

**Comment by iansk** [comment]
@alexshkatov Can you please review the material on Defender collisions?

File: rn/release-information/release-notes-20-12-update1.adoc [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+==== Defender naming scheme
+
+The Defender naming scheme has been updated to guarantee uniqueness.
+
+In AWS, the Defender name is derived as follows: `<hostname>-<instance ID>`.
+For example, `ip-172-31-80-18.ec2.internal-i-07dcaXXXXXXXXXXXX`.
+
+In Azure, the Defender name is dervived as follows: `<hostname>-<resource group name>-<subscription ID>`.
+For example, `aks-agentpool-16600000-vmss000000-MC_alexs-aks_alexs-aks_eastus-ae0XXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX`.
```
Comment (alexshkatov) [inline rn/release-information/release-notes-20-12-update1.adoc:59]: suggest to replace `alexs` with some generic name, e.g. `user` or something like this

File: rn/release-information/release-notes-20-12-update1.adoc [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+. Open the YAML file, and set the `CLOUD_HOSTNAME_ENABLED` environment variable to `true`.
++
+----
+- name: CLOUD_HOSTNAME_ENABLED
+  value: "true"
+----
+
+. Update the Prisma Cloud objects.
+
+  $ kubectl apply -f twistlock_console.yaml
```
Comment (alexshkatov) [inline rn/release-information/release-notes-20-12-update1.adoc:80]: I guess you mean `defender.yaml`

File: rn/release-information/release-notes-20-12-update1.adoc [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+Upon upgrade, old Defenders appear in Console as disconnected for some time until they're automatically removed by Console.
```
Comment (alexshkatov) [inline rn/release-information/release-notes-20-12-update1.adoc:82]: Upgraded defenders with a longer unique name appear along with the old disconnected defenders.

File: rn/release-information/release-notes-20-12-update1.adoc (hunk 1 above)
Comment (iansk) [reply rn/release-information/release-notes-20-12-update1.adoc:59]: Fixed

File: rn/release-information/release-notes-20-12-update1.adoc (hunk 2 above)
Comment (iansk) [reply rn/release-information/release-notes-20-12-update1.adoc:80]: Thanks, good catch.

File: rn/release-information/release-notes-20-12-update1.adoc (hunk 3 above)
Comment (iansk) [reply rn/release-information/release-notes-20-12-update1.adoc:82]: Thanks for the clarification. This is fixed now.

**Comment by iansk** [comment]
@alexshkatov Thanks for the review!

**Pull request merged**